pkgs/servers/monitoring/grafana/default.nix

```diff
@@ -56,14 +57,16 @@ buildGoModule rec {
     dontInstall = true;
     dontFixup = true;
     outputHashMode = "recursive";
-    outputHash = "sha256-QdyXSPshzugkDTJoUrJlHNuhPAyR9gae5Cbk8Q8FSl4=";
+    outputHash = if stdenv.isLinux then "sha256-3CZgs732c6Z64t2sfWjPAmMFKVTzoolv2TwrbjeRCBA=" else "sha256-NKEajOe9uDZw0MF5leiKBIRH1CHUELRho7gyCa96BO8=";
```

Comment:
The if/else here is wrong because we don't just have Linux or Darwin (see `lib.platforms`) even though they're less well-known.

We should either do an if/else if for `stdenv.isLinux`/`stdenv.isDarwin` and throw on else if we're sure that for each Darwin and Linux the hash is the same (please check that then).
The other option would be to specify hashes explicitly per platform:
```
hash = {
  x86_64-linux = "hash1";
  aarch64-darwin = "hash2";
  aarch64-linux = "hash3";
}.${system} or throw "Unsupported system";
```

This has the downside that we'll duplicate hashes for Linuxes though.

In any case, please make sure that we don't allow any platform in `meta.platforms` that's excluded / unsupported here.

Reply:
https://github.com/NixOS/nixpkgs/pull/297968/commits/6df2921ccaa8d02505a77978a5e6f5c6cbe052b3